prql-compiler/README.md

```diff
@@ -1,20 +1,51 @@
-# PRQL compiler
+# Rust PRQL compiler
```

Comment:
Maybe just:

```suggestion
# PRQL compiler
```

As there's only one? Or if you think the currently proposed version is clearer then fine to leave